### DISCUSSION TITLE
Clickable text that change text words

### DISCUSSION THREAD

**Discussion by og900aero** [polybar/polybar#2838] [Q&A] [answered]
I want battery module. When discharging show percentage, when left click show percentage and time, when right click show again only percentage. I tried this, but doesnt work:

label-discharging = %{A1:...:}%{A2:...:}%percentage%%%{A} %percentage% %time%{A}%percentage%%

This alway show all texts

**Comment by patrick96** [comment, marked as answer]
This is not possible. Action tags (`%{A...}`) are used for something else; they define an area that, when clicked (or scrolled over), execute some command in the background.
